tox.ini

```diff
@@ -33,6 +33,7 @@ deps =
      -r{toxinidir}/requirements_test.txt
 commands =
     python -m script.gen_requirements_all validate
```

Comment:
circle uses different way, does it need change?

Reply:
Yeah, we are importing from `script.hassfest` and so with `python -m` we set the path correctly.